demisto_sdk/commands/common/errors.py

```diff
@@ -2545,6 +2563,49 @@ def modeling_rule_keys_are_missing():
         return "The 'rules' key or the 'schema' key is missing from the modeling rule yml file. " \
                'Make sure to add them to your yml file with an empty string as value.'
 
+    @staticmethod
+    @error_code_decorator
+    def modeling_rule_schema_types_invalid(invalid_types: list):
+        return f"The following types in the schema file are invalid {','.join(invalid_types)}. " \
+               f"Valid types are: string, int , float, datetime, boolean."
+
+    @staticmethod
+    @error_code_decorator
+    def correlation_rules_files_naming_error(invalid_files: list):
+        return f"The following correlation rules files do not match the naming conventions: {','.join(invalid_files)}." \
+               f" Files in the modeling rules directory must use the pack's name as a prefix, e.g. `myPack-report1.yml`"
+
+    @staticmethod
+    @error_code_decorator
+    def xsiam_report_files_naming_error(invalid_files: list):
+        return f"The following xsiam report files do not match the naming conventions: {','.join(invalid_files)}." \
+               f"XSIAM reports title must use the pack's name as a prefix, e.g. `myPack-report1.yml`"
+
+    @staticmethod
+    @error_code_decorator
+    def parsing_rules_files_naming_error(invalid_files: list):
+        return f"The following parsing rules files do not match the naming conventions: {','.join(invalid_files)}." \
+               f" Files in the parsing rules directory must be titled exactly as the pack, e.g. `myPack.yml`."
+
+    @staticmethod
+    @error_code_decorator
+    def xdrc_templates_files_naming_error(invalid_files: list):
+        return f"The following xdrc templates do not match the naming conventions:: {','.join(invalid_files)}." \
+               f" Files in the xdrc templates directory must be titled exactly as the pack, e.g. `myPack.yml`."
+
+    @staticmethod
+    @error_code_decorator
+    def xsiam_dashboards_files_naming_error(invalid_files: list):
+        return f"The following XSIAM dashboards do not match the naming conventions:: {','.join(invalid_files)}." \
+               f" Files titles in the XSIAM dashboards directory must use the pack's name as a prefix, " \
```

Comment:
what are titles?

Reply:
The actual name of the file.

Reply:
is it called `title` elsewhere? if not, let's rename it to `file name`

Reply:
done